configs/top_down/resnet/coco/res50_coco_256x192.py

```diff
@@ -117,7 +117,9 @@
 
 data_root = 'data/coco'
 data = dict(
-    samples_per_gpu=64,
+    samples_per_gpu_train=64,
```

Comment:
Implement samples_per_gpu as videos_per_gpu here

https://github.com/open-mmlab/mmaction2/blob/master/configs/localization/bsn/bsn_tem_400x100_1x16_20e_activitynet_feature.py#L56-L60

Reply:
And I guess we have already support this syntax. (Maybe all you need to do is to write config in this format)